### PR TITLE
Adding new swag opportunities.

### LINKED ISSUE #755: Xamarin Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Xamarin (Microsoft)

Q. Where have you seen the announcement (URL)?  
A. : https://devblogs.microsoft.com/xamarin/hacktoberfest-xamarin-community-toolkit/

Q. What kind of swag (clothing, stickers, ...)?  
A. : undisclosed

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### LINKED ISSUE #768: Zup IT Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Zup IT

Q. Where have you seen the announcement (URL)?  
A. : http://insights.zup.com.br/hacktoberfest

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing

Q. Have you received swag from this?  
A. : 

Q. How long did it take?  
A. : 

Q. Does it ship worldwide?  
A. : only Brazil

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by achaljhawar**
Fixes #768
Fixes #755

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -206,6 +206,24 @@
...
+    "name": "DX Heroes",
+    "difficulty": "hard",
+    "description": "First fill the form metionedand then come in the top 3 contributors of their repo to recieve permanent badge of honour and the top contributor will recive DX heroes merch.",
+    "reference": "https://github.com/DXHeroes/dx-scanner#hacktoberfest-2020-",
+    "image": "https://raw.githubusercontent.com/achaljhawar/Hello-World/master/assets/Capture.JPG",
+    "dateAdded": "2020-10-10T12:37:23.000Z",
+    "tags": ["stickers","clothing","hacktoberfest"]
+  },
+  {
+    "name": "Earthly",
```
Comment (aslafy-z) [inline data.json:219]: This is resolved by #729

File: data.json [hunk 2]
```diff
@@ -206,6 +206,24 @@
     "dateAdded": "2019-10-27T12:37:23.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "DX Heroes",
```
Comment (aslafy-z) [inline data.json:210]: We decided not to add this one.

File: data.json [hunk 3]
```diff
@@ -718,5 +736,23 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
+    "name": "Xamarin by Microsoft",
+    "difficulty": "hard",
+    "description": "Make quality pull requests to issues that are 'approved item of work' t-shirt and other swag.",
```
Comment (aslafy-z) [inline data.json:743]: Can you cleanup a bit this description so it's clear what is required to get what swag? Thanks

File: data.json [hunk 4]
```diff
@@ -718,5 +736,23 @@
...
+    "image": "https://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2020/10/image-1-1024x536.png",
+    "dateAdded": "2020-10-10T16:35:21.000Z",
+    "tags": ["hacktoberfest", "clothing"]
+  },
+  {
+    "name": "Zup IT",
+    "difficulty": "medium",
+    "description": "Get to PRs approved in the mentioned repos and the fill a form to get awesome t-shirt delivered at your home shipping is only their in Brazil.",
+    "reference": "http://insights.zup.com.br/hacktoberfest",
+    "image": "https://d335luupugsy2.cloudfront.net/cms/files/83508/1597930522/$iw1arhcd46",
```
Comment (aslafy-z) [inline data.json:754]: use this image instead: https://i.imgur.com/eSInrEm.png

**Comment by aslafy-z** [comment]
Can you take my comments into account @achaljhawar? Thanks

**Review by aslafy-z — CHANGES_REQUESTED** [review]
Can you take my comments into account @achaljhawar? Thanks

**Comment by achaljhawar** [comment]
@aslafy-z @swapagarwal @vikaspotluri123 I think these swag opportunities aren't gonna last long as half of October is over so we should merge it as quickly as possible so that our users could get to know about them.

**Comment by aslafy-z** [comment]
You're right @achaljhawar . Can you edit do the required changes, and I'll merge this PR asap.

**Comment by achaljhawar** [comment]
@aslafy-z I am closing this pr and opening new one as I have deleted the fork

**Pull request closed without merging**